tests/access_tests.py

```diff
@@ -14,12 +14,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+# isort:skip_file
 """Unit tests for Superset"""
 import json
 import unittest
 from unittest import mock
 
-from superset import app, db, security_manager
+from tests.test_app import app  # isort:skip
```

Comment:
Why should `tests` come before `superset`?

Reply:
In this case, we need tests.test_app to load before importing downstream modules, as they end up requiring the app to have been initialized at import time. Next PR will start to clean up transitive package loads...